app/Support/ItemCollection.php

```diff
@@ -462,7 +462,8 @@ public function canShootArrows(int $min_level = 1)
             case 1:
             default:
                 return (($this->has('Bow') || $this->has('ProgressiveBow')) 
-                        && ($this->has('ShopArrow') || $this->has('SilverArrowUpgrade')))
+                        && (!$world->config('rom.rupeeBow', false)
+                            || $this->has('ShopArrow') || $this->has('SilverArrowUpgrade')))
                     || $this->has('ProgressiveBow', 2)
```

Comment:
Can you add a check for $world->config('item.overflow.count.Bow') here?  The 2nd bow might not be available.

Reply:
The 2nd Bow would actually still not allow you to shoot arrows. 
(which is a thing I didn't know about when writing that section at first)

With Progressive Bows the Behaviour is changed to always needing the ShopArrow in Retro, while the Silver Arrow Upgrade still allows for firing Silvers without it.

The Logic change for that is already included in this PR.

Reply:
Ah interesting.  I didn't know about that.

In that case, I think we need a check for shop arrows for shooting silver arrows (for basic retro beatable only).

Reply:
Added that check.